src/js/view/controls/components/chapters.mixin.js

```diff
@@ -10,6 +10,10 @@ class Cue {
     }
 
     align(duration) {
+        if (!this.time) {
+            return;
+        }
```

Comment:
Why is this necessary?

`this.time = time` is set in the constructor. You should never be instantiating cues without a time. Isn't `0` still valid?

Reply:
This was because I was calling `setCues()` wrong to test.